infrastructure/builder/src/build/index.js

```diff
@@ -1,21 +1,18 @@
-const _ = require('lodash');
 const os = require('os');
 const util = require('util');
-const fs = require('fs');
 const path = require('path');
 const config = require('typed-env-config');
 const rimraf = util.promisify(require('rimraf'));
 const mkdirp = util.promisify(require('mkdirp'));
 const {ClusterSpec} = require('../formats/cluster-spec');
 const {TerraformJson} = require('../formats/tf-json');
 const {TaskGraph, Lock, ConsoleRenderer, LogRenderer} = require('console-taskgraph');
-const {gitClone} = require('./utils');
 const generateRepoTasks = require('./repo');
 const generateMonoimageTasks = require('./monoimage');
 
-const _kindTaskGenerators = {
+const kindTaskGenerators = {
```

Comment:
Should we just get rid of this if it doesn't return anything anymore?

Reply:
It does - the functions collected in this object are used below (see line 60)